plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcConfig.java

```diff
@@ -44,6 +45,7 @@ public class BaseJdbcConfig
     private Duration metadataCacheTtl = new Duration(0, SECONDS);
     private Optional<Duration> schemaNamesCacheTtl = Optional.empty();
     private Optional<Duration> tableNamesCacheTtl = Optional.empty();
+    private Optional<Duration> statisticsCacheTtl = Optional.empty();
```

Comment:
Why not enable this by default with some value like 5m ? We enabled it by default in hive

Reply:
backwards compatibility. enabling stats cache may harm some workloads. like staging tables that are sometimes empty and sometimes full of data.

Reply:
If it's going to be better for the vast majority of workloads, then I would still enable it despite the harm to some less common cases. This can be explicitly disabled in the catalogs where it turns out to be harmful.
Alternatively, can we enable it for connectors where staging tables are rarely or not used at all ?

Reply:
I don't know the usage patterns, so I didn't want to enable it by default, at least just yet.
Do you see a problem with this PR if this is not enabled by default from the start?

Reply:
It's not a hard blocker for the PR, but the whole point of this change in hive connector was to allow us to enable stats metadata caching by default rather than just allowing for greater manual tweaking of cache ttls. If we don't have a path way to enabling this by default ever in JDBC connectors, then we're missing out on the main benefit of this change. 
Unless you have a different plan to find out the usage patterns, the only way I see of discovering the problems with enabling this by default is to just enable it and see what kind of problems are reported due to it.

Reply:
I am not against enabling it by default and I think we agree that this is not required for this PR. In other words, can be enabled as a follow-up.